java/lang/os_command_injection.yml

```diff
@@ -0,0 +1,65 @@
+imports:
+  - java_shared_lang_instance
+patterns:
+  - pattern: |
+      $<RUNTIME_INIT>.exec($<DYNAMIC_INPUT>);
+    filters:
+      - variable: RUNTIME_INIT
+        detection: java_lang_os_command_injection_runtime
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
+  - pattern: |
+      $<PROCESS_BUILDER_INIT> $<_> = new $<PROCESS_BUILDER_INIT>($<DYNAMIC_INPUT>);
+    filters:
+      - variable: PROCESS_BUILDER_INIT
+        detection: java_lang_os_command_injection_process_builder
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
+  - pattern: |
+      $<PROCESS_BUILDER_INIT>.command($<DYNAMIC_INPUT>);
+    filters:
+      - variable: PROCESS_BUILDER_INIT
+        detection: java_lang_os_command_injection_process_builder
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
+auxiliary:
+  - id: java_lang_os_command_injection_runtime
```

Comment:
Looking at the docs, I think there should be two cases for this rule:
- you have an existing instance (using `java_shared_lang_instance`)
- you called the static method `Runtime.getRuntime()`